app/services/to_form/note_mapper.rb

```diff
@@ -6,10 +6,6 @@ class NoteMapper
     ABSTRACT_TYPE = 'abstract'
     CITATION_TYPE = 'preferred citation'
 
-    def self.call(...)
-      new(...).call
-    end
-
```

Comment:
Dead code.

Reply:
I'd leave it in for the reasons previously noted.

Reply:
I'm still inclined to kill it until we need it. The `NoteMapper` wants to be opinionated about whether it's being invoked to map an abstract or a citation, so I'm not sure I see a near-term future where it'll need `.call`.